### PR TITLE
Update TPM clone command to specific commit

### PR DESCRIPTION
**Pull request by dmcilvaney**
The reference implementation is being updated regularly, indicate that the TPM is an optional component. Also indicate a known good commit to checkout.

### CODE REVIEW

File: Documentation/build-arm64-firmware.md [hunk 1]
```diff
@@ -30,10 +30,14 @@ This document describes how to set up a build environment to build the latest fi
     $ git clone --recursive -b imx_v2018.03_4.14.62_1.0.0_beta https://github.com/ms-iot/u-boot.git
     $ git clone -b codeaurora/imx_4.14.62_1.0.0_beta https://github.com/ms-iot/optee_os.git
     $ git clone --recursive https://github.com/ms-iot/mu_platform_nxp
-    $ git clone --recursive https://github.com/Microsoft/ms-tpm-20-ref
     $ git clone -b imx_4.14.62_1.0.0_beta https://source.codeaurora.org/external/imx/imx-atf
     $ git clone -b imx_4.14.62_1.0.0_beta https://source.codeaurora.org/external/imx/imx-mkimage
     ```
+    Optionally, clone the TPM reference implementation (`mu_platform_nxp` includes a copy of the TPM already)
```
Comment (hbeberman) [inline Documentation/build-arm64-firmware.md]:
```suggestion
    Optionally, clone the TPM reference implementation (`mu_platform_nxp` includes a precompiled TPM binary)
```

File: Documentation/build-firmware.md [hunk 2]
```diff
@@ -31,7 +31,11 @@ This document describes how to set up a build environment to build the latest fi
     $ git clone --recursive -b tcps-feature https://github.com/Microsoft/RIoT.git
     $ git clone -b imx https://github.com/ms-iot/imx-edk2-platforms.git
     $ git clone --recursive https://github.com/tianocore/edk2
+    ```
+    Optionally, clone the TPM reference implementation (`imx-edk2-platforms` includes a copy of the TPM already)
```
Comment (hbeberman) [inline Documentation/build-firmware.md]:
```suggestion
    Optionally, clone the TPM reference implementation (`imx-edk2-platforms` includes a precompiled TPM binary)
```

File: Documentation/build-arm64-firmware.md [hunk 3]
```diff
@@ -30,10 +30,14 @@ This document describes how to set up a build environment to build the latest fi
...
+    ```bash
+    $ git clone --recursive https://github.com/Microsoft/ms-tpm-20-ref
+    $ pushd .; cd .\ms-tpm-20-ref; git checkout 65b65354c6cce3212d9c512ec3ae2e23fe37c94d; popd
```
Comment (hbeberman) [inline Documentation/build-arm64-firmware.md]:
```suggestion
    $ pushd ms-tpm-20-ref; git checkout 65b65354c6cce3212d9c512ec3ae2e23fe37c94d; popd
```

File: Documentation/build-firmware.md [hunk 4]
```diff
@@ -31,7 +31,11 @@ This document describes how to set up a build environment to build the latest fi
...
+    ```bash
     $ git clone --recursive https://github.com/Microsoft/ms-tpm-20-ref
+    $ pushd .; cd .\ms-tpm-20-ref; git checkout 65b65354c6cce3212d9c512ec3ae2e23fe37c94d; popd
```
Comment (hbeberman) [inline Documentation/build-firmware.md]:
```suggestion
    $ pushd ms-tpm-20-ref; git checkout 65b65354c6cce3212d9c512ec3ae2e23fe37c94d; popd
```

**Review by hbeberman — CHANGES_REQUESTED** [review]

**Comment by neilsh-msft** [comment]
Can you update the arm64 documentation as well?

**Comment by dmcilvaney** [comment]
We already updated build-arm64-firmware.md, unless there is somewhere else I'm missing.

**Review by neilsh-msft — APPROVED** [review]

**Pull request merged**